core/src/main/scala/chisel3/experimental/Trace.scala

```diff
@@ -6,8 +6,19 @@ import firrtl.AnnotationSeq
 import firrtl.annotations.{Annotation, CompleteTarget, SingleTargetAnnotation}
 import firrtl.transforms.DontTouchAllTargets
 
-/** The util that records the reference map from original [[Data]]/[[Module]] annotated in Chisel and final FIRRTL. */
-object trace {
+/** The util that records the reference map from original [[Data]]/[[Module]] annotated in Chisel and final FIRRTL.
+  * @example
+  * {{{
```

Comment:
Nice example, but isn't it missing a call to `traceName`?

Reply:
oh my fault.